CONTRIBUTING.md

```diff
@@ -84,7 +84,7 @@ While you're working away in your branch it's quite possible that your upstream/
 1. [Stash](http://progit.org/book/ch6-3.html) any un-committed changes you need to
 1. `git checkout master`
 1. `git pull upstream master`
-1. `git checkout myBranch`
+1. `git checkout -b myBranch`

```

Comment:
`git checkout -b` creates a branch, but in this case the branch already exists. This section only describes how to rebase it on top of `master`.